MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```

Comment:
can you please merge in develop to this branch and ensure the date here is April 2021? 

Thank you, looks good otherwise.

Reply:
@iCollin Thank you. Done in f5c56f3e4cc6c6735fb26ca239a895be4393a969